### PR TITLE
Centralize command_line documentation

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

This PR brings the seperate `command_line` platform pages together and moves them to the integration page.
- Configuration and examples are moved.
- Config options are sorted.
~~- The `scan_interval` parameter was removed from the documentation as the current code does not support this, and a note was added this interval is 60 seconds and cannot be changed.~~

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/command_line.markdown [hunk 1]
```diff
@@ -63,25 +75,233 @@ value_template:
   description: Defines a [template](/docs/configuration/templating/#processing-incoming-data) to extract a value from the payload.
   required: false
   type: string
-scan_interval:
```
Comment (jbouwh) [inline source/_integrations/command_line.markdown:66]: This is not supported

File: source/_integrations/command_line.markdown [hunk 2]
```diff
@@ -63,25 +75,233 @@ value_template:
...
+additional benefit, if a user wanted to change the `friendly_name` / display
+name (e.g., from "Kitchen Lightswitch" to "Kitchen Switch" or
+"Living Room Light", or remove the `friendly_name` altogether), they could
+do so without needing to change existing automations or API calls.
+See aREST device below for an example.
+
 ## Execution
 
-The `command` is executed within the [configuration directory](/docs/configuration/).
+The `command` is executed within the [configuration directory](/docs/configuration/). The polling interval is 60 seconds and not configurable.
```
Comment (jbouwh) [inline source/_integrations/command_line.markdown]: Here the default polling interval is mentioned.

**Comment by jbouwh** [comment]
~~I am not sure about the scan_interval setting this one to draft~~, re-added it as it was not stale

**Review by frenck — APPROVED** [review]
Nice @jbouwh!

One of the very few still around in this old style. Thanks for this!

../Frenck

**Pull request merged**